docs/powergate/index.md

```diff
@@ -69,41 +69,51 @@ You can build and install the Powergate CLI from the [Powergate Repo](https://gi
 ```bash
 git clone git@github.com:textileio/powergate.git
 cd powergate
-make build
+make install-pow
 ```
 
 !!!info
     To compile from source, verify you have Go 1.14 or newer installed. All `make` commands install binaries in `$GOPATH/bin`, which usually is in `$PATH`, so you can run them right away from any folder in your terminal.
 
 **Using the CLI**
 
-Powergate CLI commands are just `pow`.
+You can view all the commands by running `pow --help`.
 
-![](images/powergate/../../../images/powergate/help.png)
+```bash
+$ pow --help
+A client for storage and retreival of powergate data
+
+Usage:
+  pow [flags]
+  pow [command]
+
+Available Commands:
+  admin        Provides admin commands
+  config       Provides commands to interact with cid storage configs
+  data         Provides commands to interact with general data APIs
+  deals        Provides commands to view Filecoin deal information
+  help         Help about any command
+  id           Returns the storage profile id
+  storage-jobs Provides commands to query for storage jobs in various states
+  version      Display version information for pow and the connected server
+  wallet       Provides commands about filecoin wallets
+
+Flags:
+  -h, --help                   help for pow
+      --serverAddress string   address of the powergate service api (default "127.0.0.1:5002")
+  -t, --token string           storage profile auth token
+  -v, --version                display version information for pow and the connected server
+
+Use "pow [command] --help" for more information about a command.
+```
 
 ### Multi-tiered storage
 
-The workhorse of APIs in the Powergate is called, the [FFS](ffs.md) (Filecoin File System). This module provides a multi-tiered file storage API built on Filecoin and IPFS. Storing data on IPFS and Filecoin is as easy as expressing your desired configuration for storing a Cid.
```

Comment:
I think this came from the Readme, maybe we should make the Powergate readme more succinct and avoid duplication? Looks like docs is the right place for all this.